packages/relay/src/lib/eth.ts

```diff
@@ -566,14 +571,18 @@ export class EthImpl implements Eth {
         receiptResponse.max_fee_per_gas === undefined || receiptResponse.max_fee_per_gas == '0x'
           ? receiptResponse.gas_price
           : receiptResponse.max_fee_per_gas;
+      const createdContract =
+        receiptResponse.created_contract_ids.length > 0
+          ? EthImpl.prepend0x(ContractId.fromString(receiptResponse.created_contract_ids[0]).toSolidityAddress())
```

Comment:
Is it a safe assumption for pull from index 0?
What if the createdContract isn't in that index?

You should be able to use `receiptResponse.contract_id` instead

Reply:
There is a deeper issue with detecting "contract create" transacitons with mirror nodes I need to open a bug for.  It's the best we can do in the interim, we can't distinguish null 'to' addresses because mirror node populates it with the created contract.

Reply:
Okay.
I was concerned about cases where there's multiple contacts created. 0 might not be the primary contract.
Fine for now I guess